motimo-api/src/main/java/kr/co/api/goal/GoalController.java

```diff
@@ -0,0 +1,38 @@
+package kr.co.api.goal;
+
+import kr.co.api.goal.docs.GoalControllerSwagger;
+import kr.co.api.goal.rqrs.*;
+import kr.co.api.security.annotation.AuthUser;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+@RestController
+@RequestMapping("/v1/goals")
+public class GoalController implements GoalControllerSwagger {
+
+    @ResponseStatus(HttpStatus.CREATED)
+    @PostMapping
+    public GoalIdRs create(@RequestBody GoalCreateRq rq) {
+        return new GoalIdRs(UUID.randomUUID().toString());
+    }
+
+    @PutMapping("/{id}")
+    public void update(@PathVariable String id, GoalUpdateRq rq) {
+    }
+
+    @GetMapping
+    public GoalListRs readList(@AuthUser UUID userId) {
```

Comment:
```suggestion
    public GoalListRs getGoals(@AuthUser UUID userId) {
    또는
    public GoalListRs getGoalList(@AuthUser UUID userId) {
```

요거 readList는 어떤 것을 가져오는지 명확하지 않을 수 있을거 같아요~

➕  `read`는 파일이나 스트림 같은 것들을 읽을 때 사용하는 표현인거 같아서 `get` 을 사용하도록 하는건 어떨까요?

Reply:
오호 확인했습니다~
Rs같은 경우도 `GoalGetRs`로 바꾸는 게 좋을까요?

Reply:
Response 같은 경우는 `Get` 같은건 굳이 안 넣어도 될거 같아요!
- 어떤 값을 반환하는지만?!